docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,98 @@
+            - get the coefficients for the CAP-f-T curve (IP units) using a table lookup for table J-6.  This assumes that the lookup function will return a list with the coefficients in order (C1,C2,...): `cap_f_t_coefficients = table_J_2_lookup(curve_set,"Cap-f-T")`
+            - get the coefficients for the PLR curve (IP units) using a table lookup for table J-6.  This assumes that the lookup function will return a list with the coefficients in order (C1,C2,...): `plr_coefficients = table_J_2_lookup(curve_set,"EIR-f-PLR")`
+
+            - create a variable (capacity_validation_status) to track whether the capacity validation passes and set it to "PASS" - if the capacity validation fails at any point, this variable will change to give the values at which the capacity validation failed: `capacity_validation_status = "PASS"`
+            - create a variable (power_validation_status) to track whether the power validation passes and set it to "PASS" - if the power validation fails at any point, this variable will change to give the values at which teh power validation failed: `power_validation_status = "PASS"
+
+            - convert the chiller.capacity_validation_points to a dictionary with the keys being a list of two values - [chilled_water_supply_temperature, condenser_temperature], and values being the capacity_validation_point itself.  Sorting the validation points will allow the calculations that follow to be much simpler.  Start by creating a dict capacity_validation_pts_dict: `capacity_validation_pts_dict = {}`
+            - look at each value in chiller.capacity_validation_points: `for capacity_validation_point in chiller.capacity_validation_points:`
+                - add the point to capacity_validation_pts_dict: `capacity_validation_pts_dict[[capacity_validation_point.chilled_water_supply_temperature, capacity_validation_point.condenser_temperature]] = capacity_validation_point`
+            
+            - do the same conversion with the chiller.power_validation_points and storing the results in power_validation_pts_dict.  However in this case, the values are a list of points because we expect different loads at each point: `power_validation_pts_dict = {}`
+            - look at each value in chiller.power_validation_points: `for power_validation_point in chiller.power_validation_points:`
+                - set the default to a blank list: `power_validation_pts_dict.setdefault([[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]], [])`
+                - append the power validation point to the list at this given: `power_validation_pts_dict[[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]].append(power_validation_point)`
```

Comment:
I am having a little trouble following what is being done here. Shouldn't load be another point to form the key? and then the value for the CHW, CW, and Load key would be the result?

Reply:
the values in `power_validation_pts_dict` are lists of power validation points, at the given conditions and at different loads.